Misc/NEWS

```diff
@@ -317,6 +317,9 @@ Extension Modules
 Library
 -------
 
+- bpo-30149: inspect.signature() now supports partialmethod without explicit 
+             self parameter. Patch by Dong-hee Na.
```

Comment:
This is very unusual indentation. Make the entry conforming the style of other entries.

And please use two spaces after a sentence ending period.